rustfmt.toml

```diff
@@ -0,0 +1,4 @@
+max_line_width = 120
```

Comment:
This seemed to more or less approximate what was currently there. I'm open to more fiddling if you care about some of the things rustfmt has applied, though in general it defaults to the rust style guidelines.